templates/htan-synapse-sync-kms-key.yaml

```diff
@@ -0,0 +1,90 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: HTAN SynapseSync KMS Key and IAM policy
+
+Parameters:
+  AdminRoleArn:
+    Type: String
+    Description: ARN of the Administrator Role for a particular account
+
+Resources:
+  KmsDecryptPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Sid: ReadAccess
+            Action:
+              - 'kms:Describe*'
+              - 'kms:GetKeyPolicy'
+              - 'kms:GetKeyRotationStatus'
+              - 'kms:List*'
+              - 'kms:Verify'
+            Effect: Allow
+            Resource: !GetAtt KmsKey.Arn
+          - Sid: DecryptAccess
+            Action:
+              - 'kms:Encrypt'
+              - 'kms:Decrypt'
+            Effect: Allow
+            Resource: !GetAtt KmsKey.Arn
+
+  KmsKey:
+    Type: AWS::KMS::Key
+    Properties:
+      Description: !Sub '${AWS::StackName}-KmsKey'
+      KeyPolicy:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: 'Default KMS Policy that enables IAM permissions'  # https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html#key-policy-default
+            Effect: 'Allow'
+            Principal:
+              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+            Action:
+              - 'kms:*'
+            Resource: '*'
+          - Sid: "Allow administration of the key to CFN service role"
+            Effect: "Allow"
+            Principal:
+              AWS:
+                - !ImportValue
+                    'Fn::Sub': '${AWS::Region}-bootstrap-CfServiceRoleArn'
+                - !Ref AdminRoleArn
```

Comment:
Can't we do an import as well from admin saml role?

Reply:
I don't understand your question @xschildw.  Which role specifically are you talking about?  Also what would be the purpose?  pay special attention to the parameters in the config/prod/htan-synapse-sync-kms-key.yaml file and what `AdminRoleArn` resolves to.  This config give scicomp jumpcloud admins access to this key.

Reply:
@zaro0508 , you're right I missed the config part. That's the role I was referring to, you're doing the same as import without coupling the templates. Sorry for the confusion.